examples/distributed/distributed_debate.py

```diff
@@ -99,24 +108,16 @@ def run_main_process(parsed_args: argparse.Namespace) -> None:
         port=parsed_args.con_port,
         launch_server=False,
     )
-    judge_agent = judge_agent.to_dist(
-        host=parsed_args.judge_host,
-        port=parsed_args.judge_port,
-        launch_server=False,
-    )
     participants = [pro_agent, con_agent, judge_agent]
     hint = Msg(name="System", content=ANNOUNCEMENT)
-    x = None
     with msghub(participants=participants, announcement=hint):
         for _ in range(3):
-            pro_resp = pro_agent(x)
+            pro_resp = pro_agent()
             logger.chat(pro_resp)
-            con_resp = con_agent(pro_resp)
+            con_resp = con_agent()
             logger.chat(con_resp)
-            x = judge_agent(con_resp)
-            logger.chat(x)
-        x = judge_agent(x)
-        logger.chat(x)
+            judge_agent()
+        judge_agent()
```

Comment:
Is judge agent invoked twice at the end with the same system prompt?

Reply:
I will add a conclusion prompt here for the Judge. 

For now, the announcement prompt describes the workflow of the entire debate, and all agents can follow the workflow well using gpt-3.5.